guide/inheritance-wallet/backup.md

```diff
@@ -0,0 +1,254 @@
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-pdf.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-pdf@2x.png"
+   modalImage = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-pdf@2x.png"
+   alt-text = "An example of the recovery PDF file."
+   width = 1600
+   height = 800
+   modalWidth = 4050
+   modalHeight = 2168
+%}
+
+## Backup Strategy
+Alice and Bob want to ensure that they can restore the wallet, if they ever need to. They also want to make sure that their children can access the funds if needed, while maintaining security and privacy. They decide to distribute the wallet backup material such that there are two main acces routes to retrieve it. The diagram below provides a high-level overview. 
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution@2x.png"
+   modalImage = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution@2x.png"
+   mobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-mobile.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-mobile@2x.png"
+   modalImageMobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-mobile@2x.png"
+   alt-text = "A schematic illustration showing how the backup material is distributed between the parents, their children and the lawyer."
+   width = 1600
+   height = 800
+   modalWidth = 3324
+   modalHeight = 1850
+   modalWidthMobile = 1156
+   modalHeightMobile = 2936
+%}
+
+### Primary Access (Self-Recovery)
+The primary access setup makes sure that they don’t need anybody else to retrieve the backup material and recover the wallet. To achieve this, Alice and Bob distribute the material between their home safe and the password manager.
+
+#### Home Safe
+
+Bob places the following items in a tamper-evident bag and stores it in their home safe:
+
+- A USB drive that contains the recovery kit (unencrypted).
+- The printed Recovery PDF.
+- The encryption password for the recovery kit.
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-safe.png"
+   retina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-safe@2x.png"
+   mobile = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-safe-mobile.png"
+   mobileRetina = "/assets/images/guide/inheritance-wallet/wallet-backup/recovery-tool-distribution-safe-mobile@2x.png"
+   alt-text = "A diagram showing the sequence of recovery scenarios."
+   width = 682
+   height = 316
+%}
+
+
+This way, the family can access the material if needed, but any tampering would be immediately noticeable.
+
+
+#### Password Manager
```

Comment:
I would place this section before 'Home safe' just to be consistent with the illustration which mentions this first. It also explains why the encryption key is held in the safe.